pkg/chunk/cassandra/storage_client.go

```diff
@@ -134,8 +136,14 @@ func (cfg *Config) session(name string, reg prometheus.Registerer) (*gocql.Sessi
 }
 
 // apply config settings to a cassandra ClusterConfig
-func (cfg *Config) setClusterConfig(cluster *gocql.ClusterConfig) error {
-	consistency, err := gocql.ParseConsistencyWrapper(cfg.Consistency)
+func (cfg *Config) setClusterConfig(cluster *gocql.ClusterConfig, isTableSession bool) error {
+	var consistency gocql.Consistency
+	var err error
+	if isTableSession {
+		consistency, err = gocql.ParseConsistencyWrapper(cfg.TableOperationsConsistency)
+	} else {
+		consistency, err = gocql.ParseConsistencyWrapper(cfg.Consistency)
+	}
```

Comment:
Not too sure maybe worth putting those `ParseConsistencyWrapper` also in validate config. Or is the table manager always creating a session at startup anyhow?

Reply:
We generally create a session on startup so it should be ok as is.